test/e2e/haproxy_timeouts_test.go

```diff
@@ -141,12 +142,12 @@ func TestHAProxyTimeouts(t *testing.T) {
 		cmd := []string{
 			"grep",
 			"-oP",
-			"timeout\\s+" + timeout.Name + "\\s*\\K[0-9]+(?:us|ms|s|m|h|d)",
+			"timeout\\s+" + timeout.Name + "\\s*\\K" + timeout.Value,
 			"/var/lib/haproxy/conf/haproxy.config",
 		}
 		if err := podExec(t, routerPod, &stdout, &stderr, cmd); err != nil {
 			t.Errorf("Error executing %s: %v", strings.Join(cmd, " "), err)
-			t.Errorf("stderr: %v", stderr)
+			t.Errorf("stderr: %v", stderr.Bytes())
```

Comment:
The format needs to be %s, otherwise on error you get:

```console
=== CONT  TestHAProxyTimeouts
    haproxy_timeouts_test.go:149: Error executing grep-x -oP timeout\s+client\s*\K47s /var/lib/haproxy/conf/haproxy.config: command terminated with exit code 255
    haproxy_timeouts_test.go:150: stderr: [116 105 109 101 61 34 50 48 50 50 45 48 54 45 49 52 84 49 49 58 50 50 58 49 49 90 34 32 108 101 118 101 108 61 101 114
```